docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -5,7 +5,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This fall was due to change in the batch size. I changed it from 100 to 200. I did it so it is the same as in the multi gpu example. And in the multi gpu example is set it to 200 so it can be easily run on machines with 2, 4 and 8 GPUs. </p><p>Outside of this change final accuracy variability was around 0.001. </p><p> </p><p>I fixed the seeds in DALI pipelines, it was already fixed in JAX model initialization. Single gpu test gives very stable results now. </p><p>There still is some instability in the multi gpu setup but I debugged it a bit and it comes to the numerical issues in gradient calculation on multiple GPUs. I think it is stable enough for this example.</p>